static/app/views/releases/detail/overview/releaseStats.tsx

```diff
@@ -136,192 +137,200 @@ function ReleaseStats({
         </SectionContent>
       </div>
 
-      <CrashFreeSection>
-        <SectionHeading>
-          {t('Crash Free Rate')}
-          <QuestionTooltip
-            position="top"
-            title={getSessionTermDescription(SessionTerm.CRASH_FREE, project.platform)}
-            size="sm"
-          />
-        </SectionHeading>
-        {isHealthLoading ? (
-          <Placeholder height="58px" />
-        ) : (
-          <SectionContent>
-            {defined(crashFreeSessions) || defined(crashFreeUsers) ? (
-              <CrashFreeWrapper>
-                {defined(crashFreeSessions) && (
-                  <div>
-                    <CrashFree
-                      percent={crashFreeSessions}
-                      iconSize="md"
-                      displayOption={DisplayOption.SESSIONS}
-                    />
-                  </div>
+      {!organization.features.includes('release-comparison') && (
```

Comment:
This whole diff is just wrapping the block of code with a feature flag condition. No other changes made.